Dockerfile

```diff
@@ -43,7 +46,10 @@ RUN python3 -m pip install -e git+https://github.com/kytos-ng/storehouse@${branc
  && python3 -m pip install -e git+https://github.com/kytos-ng/mef_eline@${branch_mef_eline}#egg=kytos-mef_eline \
  && python3 -m pip install -e git+https://github.com/kytos-ng/maintenance@${branch_maintenance}#egg=kytos-maintenance \
  && python3 -m pip install -e git+https://github.com/amlight/coloring@${branch_coloring}#egg=amlight-coloring \
- && python3 -m pip install -e git+https://github.com/amlight/sdntrace@${branch_sdntrace}#egg=amlight-sdntrace
+ && python3 -m pip install -e git+https://github.com/amlight/sdntrace@${branch_sdntrace}#egg=amlight-sdntrace \
+ && python3 -m pip install -e git+https://github.com/amlight/scheduler@${branch_scheduler}#egg=amlight-scheduler \
```

Comment:
Thanks for pointing that out, @viniarck. In fact, there is no need for a Napp just for scheduling events like we have now. We can leverage `apscheduler` as suggested by @ajoaoff . However, since `amlight/sdntrace_cp` is using the `amlight/scheduler` [to schedule automatic traces for normal EVCs](https://github.com/amlight/sdntrace_cp/blob/master/main.py#L38) and [to schedule automatic traces for important EVCs](https://github.com/amlight/sdntrace_cp/blob/master/main.py#L51). Maybe we can proceed how it is for now and work on this for the next release (I've just mapped the issue amlight/sdntrace_cp#15 to discuss this). What do you think?

Reply:
I see, @italovalcy, I missed this event there. Thanks for mapping this, and, for sure, agreed.